Sources/Internal/LayoutItem.swift

```diff
@@ -57,6 +59,8 @@ extension LayoutItem.ItemType: Comparable {
       return lhsMonth <= rhsMonth
     case let (.monthHeader(lhsMonth), .day(rhsDay)):
       return lhsMonth <= rhsDay.month
+    case let (.monthHeader(lhsMonth), .monthFooter(rhsMonth)):
+      return lhsMonth < rhsMonth
```

Comment:
```suggestion
      return lhsMonth <= rhsMonth
```

A month header's position "is less than" / "comes before" a month footer's position even for a month header and footer in the same month.

As this is written now, `.monthHeader(2020-06) < .monthFooter(2020-06) == false`, which should not be the wrong. Let me know if this makes sense!

Reply:
Good point, I think this was leftovers from how I first positioned these!